compiler/util/ir_data.py

```diff
@@ -106,20 +106,6 @@ def HasField(self, name):  # pylint:disable=invalid-name
         """Indicates if this class has the given field defined and it is set."""
         return getattr(self, name, None) is not None
 
-    # Non-PEP8 name to mimic the Google Protobuf interface.
-    def WhichOneof(self, oneof_name):  # pylint:disable=invalid-name
-        """Indicates which field has been set for the oneof value.
-
-        Args:
-            oneof_name: the name of the oneof construct to test.
-
-        Returns: the field name, or None if no field has been set.
-        """
-        for field_name, oneof in self.field_specs.oneof_mappings:
```

Comment:
At this point I believe `oneof_mappings` is only referenced in tests, I think we can remove it from `field_specs` (and any support for it) entirely.

Reply:
Good catch.  Done.